### DISCUSSION TITLE
qsv-copy似乎能用了?

### DISCUSSION THREAD

**Discussion by ghost** [hooke007/mpv_PlayKit#306] [mpv 通用]

**Comment by hooke007** [comment]
之前也能用（但是存在色彩转换错误qsv-copy也一样），不过我现在不用intel了

**Comment by skiqing** [comment]
T.T 垃圾AMD又用不了

**Reply by hooke007** [reply]
虽然amd的编解码出了名的不堪，
但是qsv是intel的专属解码，跟amd有什么关系。。。nvidia照样用不了qsv

**Comment by skiqing** [comment]
这是用笔记本核显跑的？

**Reply by hooke007** [reply]
> 独显直连的情况下

独显直连还能用核显吗。。。
不过intel的xe演示也是8k60，只要硬件规格支持就不是问题

**Reply by hooke007** [reply]
我换之前就是intel，我的是直连之后核显彻底离线，不管是任务管理器里还是任何软件都无法启用核显加速。
